apps/opik-backend/src/main/java/com/comet/opik/api/TraceCntResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.comet.opik.api;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+
+import java.util.List;
+
+@Builder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public record TraceCntResponse(
+        List<WorkspaceTracesCnt> workspacesTracesCnt) {
+    public static TraceCntResponse empty() {
+        return new TraceCntResponse(List.of());
+    }
+
+    @Builder(toBuilder = true)
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+    public record WorkspaceTracesCnt(
```

Comment:
I believe we should favor readability in these cases 

```suggestion
    public record WorkspaceTracesCount(
```

Reply:
updated